### PR TITLE
;Update commands.js

### PR DESCRIPTION
**Pull request by MasterFloat**
Make people that have enough power to modnote but not enough power to talk during a modchat unable to modnote.

### CODE REVIEW

**Comment by MasterFloat** [comment]
What's that?

File: commands.js [hunk 1]
```diff
@@ -1352,8 +1352,11 @@ exports.commands = {
 
 	mn: 'modnote',
 	modnote: function (target, room, user, connection) {
+		var groupName = Config.groups[room.modchat].name || room.modchat;
+		
 		if (!target) return this.parse('/help modnote');
 		if ((user.locked || room.isMuted(user)) && !user.can('bypassall')) return this.errorReply("You cannot do this while unable to talk.");
```
Comment (panpawn) [inline commands.js]: Why not just add it to this check?

Comment (Zarel) [inline commands.js]: Maybe we should just run it through `canTalk` instead of all these separate checks.

Comment (panpawn) [inline commands.js]: That works too

**Comment by panpawn** [comment]
This can probably be closed from https://github.com/Zarel/Pokemon-Showdown/commit/a64875c2ed3d02aea586b5a2c8e9a8407d798716 now

**Pull request closed without merging**